### PR TITLE
Bugfix/#239 firebase configurator script

### LINKED ISSUE #239: Not having firebase plist files breaks the build
When starting a new app that does not have firebase integration initially, the builds are broken. We should try to find a way to NOT have all this setup from the get-go

### PR DESCRIPTION
**Pull request by vanlooverenkoen**
Fixed #239

### CODE REVIEW

File: tool/dart_tool/firebase_configurator.dart [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+  _removeLineInFileStartWith('android/app/build.gradle', "apply plugin: 'com.google.firebase.crashlytics'");
+  _removeLineInFileStartWith('pubspec.yaml', '  firebase_analytics');
+  _removeLineInFileStartWith('pubspec.yaml', '  firebase_core');
+  _removeLineInFileStartWith('pubspec.yaml', '  firebase_crashlytics');
+  _removeLineInFileStartWith('lib/main_common.dart', "import 'package:firebase_core/firebase_core.dart';");
+  _removeLineInFileStartWith('lib/di/injectable.dart', "import 'package:firebase_analytics/firebase_analytics.dart';");
+  _removeLineInFileStartWith('lib/di/injectable.dart', "import 'package:firebase_analytics/firebase_analytics.dart';");
+  _removeLineInFileStartWith('lib/main_common.dart', "import 'package:firebase_crashlytics/firebase_crashlytics.dart';");
+  _removeLineInFileStartWith('lib/main_common.dart', "    await _setupCrashLogging");
+  _removeLineInFileStartWith('ios/Runner.xcodeproj/project.pbxproj', "		3629EB6826E758C500F774E2 /* GoogleService-Info-dev.plist in Resources */ = {isa = PBXBuildFile; fileRef = 3629EB6726E758C500F774E2 /* GoogleService-Info-dev.plist */; };");
```
Comment (NicolaVerbeeck) [inline tool/dart_tool/firebase_configurator.dart:34]: This feels like it will break due to changing file refs

**Review by NicolaVerbeeck — CHANGES_REQUESTED** [review]
The replacement of the files in xcode is not configuration change safe I think

File: tool/rename_project.sh [hunk 2]
```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-flutter pub run tool/dart_tool/rename_project.dart
\ No newline at end of file
+dart pub run tool/dart_tool/rename_project.dart
```
Comment (ikbendewilliam) [inline tool/rename_project.sh:3]: merge conflict with use fvm branch

**Comment by NicolaVerbeeck** [comment]
Any updates on this?

**Comment by vanlooverenkoen** [comment]
We will wait on the flutter_template refactor. And see what we decide there.

**Comment by vanlooverenkoen** [comment]
Won't use this

**Pull request closed without merging**